spec/workers/sync_users_job_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe SyncUsersJob do
+  before do
+    Sidekiq::Testing.fake!
+  end
+
+  it { expect(SyncUsersJob).to be_retryable 3 }
+  it { expect(SyncUsersJob).to be_processed_in :users }
+
+  describe "#perform_async" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.